Formula/symfony.rb

```diff
@@ -0,0 +1,18 @@
+class Symfony < Formula
+  desc "Installs the Symfony framework for PHP"
+  homepage "https://symfony.com/"
+  url "https://github.com/symfony/cli/releases/download/v4.7.0/symfony_darwin_amd64.gz"
+  version "v4.7.0"
+  sha256 "c9be77cda0ee6b2f144bf8c5648a7500b01d3f4bd19d838d7a11a32be0f648b3"
+
+  depends_on "php"
+  depends_on "composer"
+
+  def install
+    bin.install "symfony" => "symfony"
+  end
+
+  test do
+    assert_match /2017-2019 Symfony SAS/, shell_output("symfony -v").strip
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (`foo --version` or `foo --help`) are not sufficient, as explained in the [formula cookbook](https://docs.brew.sh/Formula-Cookbook).

In most cases, a good test would involve running a simple test case: run `#{bin}/foo input.txt`.
- Then you can check that the output is as expected (with `assert_equal` or `assert_match` on the output of `shell_output`)
- You can also check that an output file was created, if that is expected: `assert_predicate testpath/"output.txt", :exist?`

Some advice for specific cases:
- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.